mlrun/utils/notifications/notification/base.py

```diff
@@ -28,54 +28,40 @@ class NotificationSeverity(enum.Enum):
 
 class NotificationBase:
     def __init__(
+        self,
+        params: typing.Dict[str, str] = None,
+    ):
```

Comment:
i guess we need name (optional), so in case we have few notifiers of the same kind. Also severity can be defined by the class (default) and can optionally be overwritten by the message

Reply:
I can add a default severity in general, but why the name?
If we have multiple notifiers of the same kind (say slack) and a run has some config for a slack notification. How will you decide which named notifier to use?